src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -196,3 +200,66 @@
               :on-press            (fn []
                                      (rf/dispatch [:hide-bottom-sheet])
                                      (when on-press (on-press)))}]))]])))
+
+(defn get-reaction-authors-accessibility-id
+  [reaction-type]
+  (-> reaction-type
+      constants/reactions
+      name
+      (str "-reaction-count")))
+
+(defn contact-list-item-fn
+  [{:keys [from compressed-key]}]
+  (let [[primary-name secondary-name] (rf/sub [:contacts/contact-two-names-by-identity
+                                               from])
+        {:keys [ens-verified added?]} (rf/sub [:contacts/contact-by-address from])]
+    ^{:key (str compressed-key)}
```

Comment:
Isn't `compressed-key` always a string? If this is not the case then we should actually change the place that's transforming the compressed key to something else, otherwise we can drop `str`.

Reply:
Forgot that, And for some reason I thought keys needed to strings so no more casting them.

Reply:
You're correct, AFAIR, in React keys can be numbers or strings, and that's it. Since compressed key should always be strings (not keywords) then it should be safe to use them without casting.